package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Project that contains all hosted contents of Dynamo Windows client",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run dev & npm run prod",
+    "dev": "set production_build=0 & webpack",
+    "prod": "set production_build=1 & webpack"
```

Comment:
This is great, combining `dev` and `prod` into one `build` command, while still retain the original two so that one can choose to run either `dev` and `prod` separately.